docs/ceph/ceph_rgw.md

```diff
@@ -205,7 +205,7 @@ network_config:
   - ip_netmask: {{ storage_ip }}/{{ storage_cidr }}
   routes: {{ storage_host_routes }}
 - type: ovs_bridge
-  name: {{ neutron_physical_bridge_name }}
+  name: {{ edpm_network_config_bridge_name }}
```

Comment:
This file is being passed into `openstack overcloud node provision` command. AFAICT it does not use the EDPM naming and changing the variable name here would result in breakage.